### ISSUE TITLE
NVDA does not read additional description provided using `aria-describedby ` to the grouping element in the browsed mode.

### ISSUE THREAD

**Issue by gauravahir28** [nvaccess/nvda#15856]
### Steps to reproduce:
- Download the attached file and remove ".txt" extention from the file name!
- Open the file in chrome.
-  navigate through the available elements in the brows mode!
### Actual behavior:

even though the "Please select atleast 1 hobby" error is referenced using aria-describedby, it isn't announced by the NVDA!
Whereas, if similar steps were performed using focus mode, NVDA does read the referenced text by aria-describedby!
### Expected behavior:

NVDA also reads the text referenced using aria-describedby.
**Note: this issue is specifically applicable to those elements having role="group".**
#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
NVDA 2023.3
#### Windows version:
Windows 11.
#### Name and version of other software in use when reproducing the issue:
Google chrome: 119.
Firefox: 120.
Microsoft Edge: 119
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Similar issue is applicable in the previous versions as well!
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.
### Attachment:
[NVDA_aria-describedby_issue.html.txt](https://github.com/nvaccess/nvda/files/13499899/NVDA_aria-describedby_issue.html.txt)

**Comment by Adriani90** [comment]
Closing as duplicate of #12718.
